tools/ReleaseHelpers.cs

```diff
@@ -12,12 +13,10 @@ public static ICollection<int> GetIssueNumbersReferencedFromReleases(IEnumerable
             var issuesReferencedFromRelease = new HashSet<int>();
             foreach (var release in releases)
             {
-                foreach (Match match in Regex.Matches(release.Body, @"\(\s*#(?<issueNumber>[0-9]+)(,\s*#(?<issueNumber>[0-9]+))*\s*\)"))
+                foreach (var capture in Regex.Matches(release.Body, @"\(\s*#(?<issueNumber>[0-9]+)(,\s*#(?<issueNumber>[0-9]+))*\s*\)")
+                                             .SelectMany(match => match.Groups["issueNumber"].Captures))
                 {
-                    foreach (Capture capture in match.Groups["issueNumber"].Captures)
-                    {
-                        issuesReferencedFromRelease.Add(int.Parse(capture.Value, NumberStyles.Integer, NumberFormatInfo.InvariantInfo));
-                    }
+                    issuesReferencedFromRelease.Add(int.Parse(capture.Value, NumberStyles.Integer, NumberFormatInfo.InvariantInfo));

```

Comment:
There were warnings all in here, and I noticed that VS wanted to Linq up part of the query, so I let it, and... the warnings went away!